pyaptly/util.py

```diff
@@ -28,6 +30,13 @@
 lg = logging.getLogger(__name__)
 
 
+def write_traceback():  # pragma: no cover
+    with NamedTemporaryFile("w", delete=False) as tmp:
+        tmp.write(traceback.format_exc())
+        tmp.close()
+    return tmp.name
+
```

Comment:
This is also in another PR, I assume the duplication is just for completeness?

Reply:
Yes, I had to or CI would fail. I decided to start doing such mini duplications if it allows another PR to go out